app/src/components/Button.tsx

```diff
@@ -1,32 +1,59 @@
 import { Trans } from 'next-i18next'
-import { ReactElement } from 'react'
+import React, { MouseEvent, ReactElement } from 'react'
 
-type Props = {
+import { i18nKey } from '@src/types'
+
+export const buttonStyleOptions = [
+  'default',
+  'unstyled',
+  'secondary',
+  'accent-cool',
+  'accent-warm',
+  'base',
+  'outline',
+  'big',
+]
```

Comment:
Notes for reviewers: Add support for all the different button style options in https://designsystem.digital.gov/components/button/